src/engine/source/main.cpp

```diff
@@ -343,6 +344,9 @@ void configureSubCommandCatalog(std::shared_ptr<CLI::App> app)
             "path", args::catalogPath, "Path to the directory containing the item files.")
         ->required()
         ->check(CLI::ExistingDirectory);
+    load_subcommand
+        ->add_flag(
+            "-r, --recursive", args::catalogRecursive, "Recursive loading of the directory.");
```

Comment:
if not required should it have a default value or is false taken as default?

Reply:
The `-r` option is a flag. It is used to load the decoders of a folder recursively. If this option is not set, by default it loads only the decoders of that folder in the first level.